specification/guestconfiguration/resource-manager/readme.az.md

```diff
@@ -17,5 +17,7 @@ cli:
             prop: 'properties'
           flatten: true
 
-
+        - where:
```

Comment:
What's the PR do? For cli code-gen internal test? What's the guestconfiguration?

Reply:
Hi @ruowan , we are doing the codegen validation for guestconfiguration. And I sent you a email for the background. Thank you.